src/static/index.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module '*.svg' {
```

Comment:
Had you saw some errors with types of images imports? I wonder why did you add type definitions for them.

Reply:
Indeed, I was getting errors when importing images. Error were also noticed for earlier imported images, too, so I decided to add type definitions to fix it and avoid unnecessary warnings.